spec/models/user_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe User, type: :model do
+  describe "validations" do
+    it {should validate_presence_of(:name)}
+    it {should validate_presence_of(:address)}
+    it {should validate_presence_of(:city)}
+    it {should validate_presence_of(:state)}
+    it {should validate_presence_of(:zip)}
+    it {should validate_presence_of(:email)}
+    it {should validate_uniqueness_of(:email)}
+    it {should validate_presence_of(:password)}
```

Comment:
I didn't know this is how you set up the model test for 'password', I thought it might have to be 'password_digest', good to know